modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Application.ts

```diff
@@ -48,9 +53,15 @@ export function deleteApplicationByExternalReferenceCode(
 		`${DeliveryAPI}/by-external-reference-code/${externalReferenceCode}`
 	);
 }
-const products: any = localStorage.getItem('raylife-ap-storage');
 
-const adaptToFormApplicationRequest = (state: any, status: any) => ({
+const products: string = localStorage.getItem('raylife-ap-storage') as string;
+
+const productAutoERC = 'RAYAP-001';
```

Comment:
Hey Joana, since it is mock data you can declare outside the component.

Reply:
Done!